website/src/components/Blockquote.tsx

```diff
@@ -1,21 +1,52 @@
+import classNames from 'classnames'
 import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
 import { Link } from 'gatsby'
 import React, { FunctionComponent, ReactFragment } from 'react'
 
 export const Blockquote: FunctionComponent<{
     quote: string
     by?: string | ReactFragment
-}> = ({ quote, by }) => (
-    <blockquote className="p-3 bg-light rounded rounded-lg">
-        &ldquo;{quote}&rdquo;
-        {by && (
-            <>
-                <br />
-                <div className="text-right text-muted">&mdash; {by}</div>
-            </>
-        )}
-    </blockquote>
-)
+    logoHref?: string
+    logoImage?: string
+    border?: boolean
+    headline?: string
+}> = ({ quote, by, logoHref, logoImage, border, headline }) => {
+    const quoteStyles = 'p-3 rounded rounded-lg text-center'
+
+    return (
+        <>
+            <blockquote
+                className={
+                    border
+                        ? classNames(quoteStyles, 'case-studies__quote--in-content')
+                        : classNames(quoteStyles, 'bg-light')
+                }
+            >
+                {border && headline ? (
+                    <p className="font-weight-normal">
+                        <h5 className="font-weight-bold mb-4">{headline}</h5>
+                        &ldquo;{quote}&rdquo;
+                    </p>
+                ) : (
+                    <>&ldquo;{quote}&rdquo;</>
+                )}
+                {by && (
+                    <>
+                        <br />
+                        <div className="text-center text-muted">&mdash; {by}</div>
```

Comment:
Could we make this align with the orange left border like in the design?

Reply:
Hey! So this is not the case elsewhere in the site: 

Case studies template:
<img width="671" alt="Screen Shot 2022-03-24 at 7 47 04 AM" src="https://user-images.githubusercontent.com/43454804/159910038-4794e6e6-9dc8-44b9-b227-def0882d300f.png">

Use cases homepage:
<img width="617" alt="Screen Shot 2022-03-24 at 7 47 17 AM" src="https://user-images.githubusercontent.com/43454804/159910083-d487ec39-b131-4904-97dc-488193b15a71.png">

Rather than going back and changing all of these to match the design, I would vote that the design match what's already on the site. What are your thoughts?

Reply:
Ok, no worries, let's leave it for now since it involves a lot more scope creep here. But I think in the replatform we should make use of the `BlockQuote` component everywhere, improve that component, and then that'll help improve our design consistency. 😁